package.json

```diff
@@ -1,74 +1,80 @@
 {
-  "name": "htmlhint",
-  "version": "0.0.0-development",
-  "description": "The Static Code Analysis Tool for your HTML",
-  "main": "./index",
-  "dependencies": {
-    "async": "2.6.1",
-    "colors": "1.3.2",
-    "commander": "2.17.1",
-    "csslint": "1.0.5",
-    "glob": "7.1.3",
-    "jshint": "2.9.6",
-    "parse-glob": "3.0.4",
-    "path-parse": "1.0.6",
-    "request": "2.88.0",
-    "strip-json-comments": "2.0.1",
-    "xml": "1.0.1"
-  },
-  "devDependencies": {
-    "expect.js": "0.3.1",
-    "grunt": "1.0.3",
-    "grunt-cli": "1.3.1",
-    "grunt-contrib-clean": "1.1.0",
-    "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-jshint": "1.1.0",
-    "grunt-contrib-uglify": "3.4.0",
-    "grunt-contrib-watch": "1.1.0",
-    "grunt-exec": "3.0.0",
-    "grunt-replace": "1.0.1",
-    "istanbul": "0.4.5",
-    "istanbul-coveralls": "1.0.3",
-    "mocha": "5.2.0",
-    "semantic-release": "^15.9.9"
-  },
-  "bin": {
-    "htmlhint": "./bin/htmlhint"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/thedaviddias/HTMLHint.git"
-  },
-  "bugs": {
-    "url": "https://github.com/thedaviddias/HTMLHint/issues"
-  },
-  "keywords": [
-    "html",
-    "hint",
-    "analysis",
-    "javascript"
-  ],
-  "author": {
```

Comment:
I'm missing the "author" and "contributors" :(

Reply:
Don't see why, the list of contributors is given by Github easily and everyone can see it.

Reply:
Because you can see the package.json in node_modules 
(It's only for author and main contributors)